AnaProd/tasks.py

```diff
@@ -87,12 +96,12 @@ def workflow_requires(self):
         return { "inputFile": InputFileTask.req(self) }
 
     def output(self):
-        sample_name, isData = self.branch_data
+        sample_name, isData, isPrivateNano = self.branch_data
         return self.remote_target('anaCache', self.period, f'{sample_name}.yaml', fs=self.fs_anaCache)
 
     def run(self):
-        sample_name, isData = self.branch_data
-        if isData:
+        sample_name, isData, isPrivateNano = self.branch_data
+        if isData or isPrivateNano:
```

Comment:
This logic looks wrong. We need a cache for all MC datasets, regardless of whether it is private or not.